### PR TITLE
Console server

### PR DESCRIPTION
**Pull request by anukritijha**
Moving the UI to the console-server branch of EnMasseProject.

### CODE REVIEW

File: .history/console/console-init/oauth-proxy/cfg/oauth-proxy-kubernetes_20200114170523.cfg [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+http_address = ""
+https_address = ":8443"
+provider = "oidc"
+oidc_issuer_url = "${ISSUER}"
+
+tls_cert_file = "/etc/tls/private/tls.crt"
+tls_key_file = "/etc/tls/private/tls.key"
+
+upstreams = [
+<<<<<<< HEAD
```
Comment (kornys) [inline .history/console/console-init/oauth-proxy/cfg/oauth-proxy-kubernetes_20200114170523.cfg]: seems you have unresolved merge conflicts, please fix that

Comment (anukritijha) [reply .history/console/console-init/oauth-proxy/cfg/oauth-proxy-kubernetes_20200114170523.cfg]: Okay, let me check.

Comment (anukritijha) [reply .history/console/console-init/oauth-proxy/cfg/oauth-proxy-kubernetes_20200114170523.cfg]: Updated the pull request.

**Comment by k-wall** [comment]
The PR is also adding ui-legacy again.  If easy, remove it from the PR.  Otherwise we can fix up later.

**Comment by anukritijha** [comment]
> The PR includes files in a .history directory (e.g. .history/console/console-init/oauth-proxy/cfg/oauth-proxy-kubernetes_2020011417). These are spurious.

@k-wall Removed .history files

**Pull request merged**